doc-content/enterprise-only/employee-rostering/er-individualroster-proc.adoc

```diff
@@ -0,0 +1,11 @@
+[id='er-individualroster-proc']
+= Viewing employee shifts
+
+You can view the assigned shifts for particular employees in an employee-centric table. The information is the same as the Shift Roster, but the viewing format might be more convenient for informing employees about their assigned shifts.
+
+.Procedure
+. To view a table of employees and shifts, click the *Availability Roster* tab.
+. In the top right part of the browser window, the *Week of* field lists the week for which the roster is displayed, To view other weeks, use the *<* and *>* buttons next to the *Week of* field.
```

Comment:
Sorry but no. I could agree to make this entire paragraph "not a step", but I can't break the logic here by putting a step number in the middle. Especially since there are the particular < and > buttons next to the week of field, so the broken-up version might ont make sense (there are other < and < buttons).

Reply:
Putting a note about this in BXMSDOC-3185 as we do ont have the time for this discussion right now and it might need to be wider. It is really a good question (how to apply procedure model to this reality).